plugins/tds61.js

```diff
@@ -644,6 +644,9 @@ tds61 = {
         // Drop the FCSUBTYPE since we don't use it
         if (attrs.FCSUBTYPE) delete attrs.FCSUBTYPE;
 
+        // Drop the the "Not Found" F_CODE. This is from the UI
+        if (attrs.F_CODE == 'Not found') delete attrs.F_CODE;
+
```

Comment:
This change is going to break the UI I'm afraid.  The UI code is currently looking for Feature Code = Not found to detect an out of spec / partial translation, which it currently can't do anything with and so shows an "Feature out of spec, unable to translate" error.